c-bindings/libjsonnet.cpp

```diff
@@ -34,6 +34,22 @@ struct JsonnetJsonValue* jsonnet_internal_execute_native(JsonnetNativeCallback *
     return (cb)(ctx, argv, success);
 }
 
+char* jsonnet_internal_execute_import(JsonnetImportCallback *cb,
+                                      void *ctx,
+                                      const char *base,
+                                      const char *rel,
+                                      char **found_here,
+                                      int *success)
+{
+    return (cb)(ctx, base, rel, found_here, success);
+}
+
+void jsonnet_internal_free_string(char *str) {
+    if (str != nullptr) {
```

Comment:
FYI: This `if` is not necessary. Freeing `nullptr` is a no-op (https://isocpp.org/wiki/faq/freestore-mgmt#delete-handles-null). Some implementation used to be non-compliant in the past, so this check became a thing, but I don't think it's worth doing anymore.

Reply:
Thank you, I'll keep it on mind!